src/main/resources/application.yml

```diff
@@ -13,6 +13,6 @@ logging:
       springframework:
         boot: INFO
     pl:
-      futurecollars:
+      future-collars:
```

Comment:
Powinno zostać "futurecollars", ponieważ jest to nazwa jednego z twoich pakietów (.../java/pl/**futurecollars**/invoicing/...)